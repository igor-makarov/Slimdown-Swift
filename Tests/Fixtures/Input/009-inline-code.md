More text with `inline($code)` sample.