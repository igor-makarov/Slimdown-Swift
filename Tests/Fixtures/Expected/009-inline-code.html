<p>More text with <code>inline($code)</code> sample.</p>